### PR TITLE
Updated twin_network_for_diffraction.ipynb

### PR DESCRIPTION
**Pull request by EricFlorin**
Added `load_data()` function to load and preprocess the data for model training. Added rotation, zoom, and flip transforms into addNoise(). Added cells to display model metrics on a small test set after model training was complete. Added cells to make visualization of model predictions compared to ground truth, as well as a simple demo cell to showcase model prediction capabilities with a single random diffraction image.

### CODE REVIEW

**Comment by chuckie82** [comment]
@EricFlorin I merged your changes, but I have couple of comments:
1) It's very difficult to maintain code that has "uncomment for shawn's thing and uncomment here for eric's thing". We should uncomment and test the code or remove entirely
2) It's bad practice to put your personal paths in the code. 
There's a dataset folder under /reg/data/ana03/scratch where we should put all the training dataset.
